algorithms/C/strings/reverse a string with spaces using 2d arrays

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<string.h>
+int main()
+{
+    char s1[100][100],s[100];
+    int k=0,j=0,i,n;
+    printf("Enter the string: ");
+    gets(s);                                   // K DENOTES NUMBER OF WORDS
```

Comment:
the comments looks misplaced to me, are you sure you placed at correct line ( there seem to be no context of `J` or `K` in the aforementioned lines.